metadata-ingestion/src/datahub/ingestion/source/sql/trino.py

```diff
@@ -175,6 +236,68 @@ def get_db_name(self, inspector: Inspector) -> str:
         else:
             return super().get_db_name(inspector)
 
+    def _get_sibling_urn(
+        self,
+        dataset_name: str,
+        inspector: Inspector,
+        schema: str,
+        table: str,
+    ) -> Optional[str]:
+        catalog_name = dataset_name.split(".")[0]
+        connector_platform_details: Optional[PlatformDetail] = None
+
+        connector_name = get_catalog_connector_name(catalog_name, inspector)
+        if connector_name:
+            connector_platform_details = KNOWN_CONNECTOR_PLATFORM_MAPPING.get(
+                connector_name
+            )
+
+        connector_details = self.config.catalog_to_connector_details.get(
+            catalog_name, ConnectorDetail()
+        )
+
+        if connector_platform_details:
```

Comment:
why can't we just always try to generate this relationship?

Reply:
Because there can be some trino connector platform which datahub doesn't support yet.
Eg: https://trino.io/docs/current/connector/hudi.html